java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -1485,6 +1488,18 @@ public Predicate<? super T> getFilter() {
         return filter;
     }
 
+    static class DateRenderer extends DefaultTableCellRenderer {
+        @Override
+        public Component getTableCellRendererComponent( JTable table, Object value,
+            boolean isSelected, boolean hasFocus, int row, int column) {
+            JLabel c = (JLabel) super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
```

Comment:
This would be slightly faster as an `if {} else {}`, but it's not a big deal.

Reply:
I'm not 100% sure what the else would be here as I'm keen on maintaining existing table cell formattings, font size, background colour for row selected etc.

I've found another mini tweak however inside that method by using a static DateFormat, rather than creating a new one for each method call.

Reply:
Ah, I see your point about the basic formatting of fhe JLabel.  I missed that.  Thanks! Sorry for  my confusion.